app/services/EventSyncService.ts

```diff
@@ -0,0 +1,68 @@
+import { dataRealmCommon } from './../database/dbquery/dataRealmCommon';
+import analytics from '@react-native-firebase/analytics';
+
+
```

Comment:
Can we document methods to understand its purpose? For example: "A method to log events to Firebase Analytics".
"A method storing unsynchronized events in a local database Realm", "A method synchronizing events when the device is online." etc.